isos_post.py

```diff
@@ -82,9 +85,9 @@ def set_job_priority(host, job_id, priority):
                             'wicked_startandstop': 'wicked_startandstop_ref,wicked_startandstop_sut',
                             'wicked_aggregate': 'wicked_aggregate_ref,wicked_aggregate_sut'}
     if (args.alias == 'list'):
-        print('Aliases:')
+        logging.debug('Aliases:')
```

Comment:
Not sure if we should use `logging` here. Doesn't have a logger a time prefix and some other information which we do not need here?

Reply:
I added special formatting as for now it is ok